refinery/ui/source/js/data-set-about/ctrls/groups-perms-ctrl.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Switched over the share_component to group_perms_component name, due to new apis.